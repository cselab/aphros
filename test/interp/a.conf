


set string c   a b   s c    

set string   b    asdf  
del b
set int   b    123  


set double a   1  


set double a   2.9   
set vect   f   1   2 34   5.


