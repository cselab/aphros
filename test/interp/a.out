string c = a b   s c    
int b = 123
double a = 2.9
vect f = 1 2 34 5 
